source/jormungandr/jormungandr/interfaces/v1/fields.py

```diff
@@ -536,7 +544,7 @@ def format(self, value):
 
 disruption_severity = {
     "name": fields.String(),
-    "effect": fields.String(),
+    "effect": enum_type(attribute='effect', lower_case=False),
```

Comment:
Why do you have to mention attribute? Not sure, but isn't that automagic?

Reply:
It's not necessary but i added that to distinct from stop_time_effect in my modifications